var/spack/repos/builtin/packages/py-crcmod/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyCrcmod(PythonPackage):
+    """Python module for generating objects that
+       compute the Cyclic Redundancy Check (CRC)"""
+
+    homepage = "https://github.com/gsutil-mirrors/crcmod"
```

Comment:
When I visit this link it includes a message:

> Unofficial mirror of crcmod (https://bitbucket.org/cmcqueen1975/crcmod). This repository may disappear, so do not depend on it.

so I was wondering why not use *that* link, but when I try it, bitbucket tells me there's no such project. Do you have a notion of which repo is "official"?

Reply:
Thanks for pointing it out. I updated the homepage to what they have in PyPi.